en_us/install_operations/source/installation/devstack/install_devstack.rst

```diff
@@ -149,7 +149,7 @@ client, follow these steps.
 
    .. code-block:: bash
 
-     curl -O https://raw.github.com/edx/configuration/master/vagrant/release/devstack/Vagrantfile
+     curl -L https://raw.github.com/edx/configuration/master/vagrant/release/devstack/Vagrantfile > Vagrantfile
```

Comment:
It seems that the improved command should be:

``curl -LO https://raw.github.com/edx/configuration/master/vagrant/release/devstack/Vagrantfile``